server/src/com/cloud/network/vpn/RemoteAccessVpnManagerImpl.java

```diff
@@ -596,7 +597,7 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
 
 
         sb.and("id", sb.entity().getId(), SearchCriteria.Op.EQ);
-        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.EQ);
+        sb.and("username", sb.entity().getUsername(), SearchCriteria.Op.LIKE);
```

Comment:
@syed Used LIKE instead of EQ as username will be used in a WHERE clause to search for a specified pattern in a column using the keyword.